actions/epic_create.yaml

```diff
@@ -20,7 +20,7 @@ parameters:
     position: 1
   labels:
     description: "The comma-separated list of labels"
```

Comment:
Est-ce que ce serait pas plutôt du coup `An array of labels` ? si c'est pu une string?